### DISCUSSION TITLE
Know our community

### DISCUSSION THREAD

**Discussion by s-leroux** [EddieHubCommunity/support#4027] [Ideas]
EddieHub does not collect user information. We should be proud of that. Nevertheless, it would be interesting to have some demographic portrait of our members. It could be useful internally at EddieHub to develop relevant partnerships or raise sponsors. It might also be useful for other members like content creators that could adjust their work to match the need of the community.

The poll could be run/sampled at regular intervals to show the demographic evolution of the community over time.

In this discussion, I target technically-related information like the development environment, the area of interest, or the level of experience of the member. *Not* socio-economic or personal data.

I see two ways we could gather that information:

## Options 1:
Using Discord poll like [I tried here](https://discord.com/channels/699608417039286293/699608417928740944/980207096144461824).It was inspired by [this message](https://discord.com/channels/699608417039286293/699609729353384036/979058265440808960). I don't know if this is standard practice on Discord but I found the idea good.

With this option, we would end up with one post per question. Answer who wants and when they want.

### Pros:
- This allows for multiple answers per question. 
- This also allows people to change their vote at any time.
- Once set up, it is entirely passive. We just take snapshots of the votes at regular intervals.
### Cons:
- The votes are not anonymous, since you can see who's voted what by a long click on the icons
- Accessibility?
- When posted on an open channel as I did, the polls are rapidly buried by other messages. A locked dedicated channel might be to consider (unless discord provides better solutions?)

## Option 2:
Once or two times each year, we launch an official consultation using Google Forms or Survey Monkey.
The link to the survey could either be published on the discord server or sent by mail to the members.

### Pros:
- You reach people that aren't active on Discord, or that simply have forgotten about the evergreen poll.
- Could be made really anonymous
- Text-only so more accessible
### Cons:
- Seems to be more work (unless someone with experience in this sort of thing contradicts me)

In all cases, it should be clear that participation is voluntary.

**Comment by SaraJaoude** [comment]
This is an interesting idea @s-leroux 

- Recognising that there are a number of EddieHub community members that do not wish to use Discord (for whatever reason) or are not active as you say, your Option 2 may be able to have greater reach.  
- Rather than a specific mail-out we could potentially include a link to the survey on the EddieHub monthly Newsletter as well as as a header on Discord to maximise reach
- The platform I use to produce the Newsletter also allows me to create surveys which can be made anonymous 
- Do you think you might be able to share the questions from your poll here so that we have this within the Discussion thread?

**Reply by s-leroux** [reply]
@SaraJaoude, everyone,

I posted below the questions I had drafted:
- [What is your platform of development?](https://github.com/EddieHubCommunity/support/discussions/4027#discussioncomment-2847285)
- [Which affirmation describes you the best?](https://github.com/EddieHubCommunity/support/discussions/4027#discussioncomment-2847291)
- [What are your main areas of interest in IT](https://github.com/EddieHubCommunity/support/discussions/4027#discussioncomment-2847296)

Each question is posted in its own "answer", post your comments in the relevant "reply" text area so things will stay organized.

**Comment by s-leroux** [comment]
## What is your platform of development?
[multiple answers possible]
-  Linux [should we include WSL here?]
-  *BSD 
-  Windows 
-  Mac
-  Other (specify)

**Comment by s-leroux** [comment]
## Which affirmation describes you *the best*?
-  I study IT as part of my school curriculum
-  I study IT by myself
-  I'm looking for a job in IT
-  I work in IT
-  I'm retired from IT
-  I practice IT as a hobbyist
-  I create content (blog, video) about IT
-  None of the above

**Comment by s-leroux** [comment]
## What are your main areas of interest in IT
[multiple answers possible -- limit to 3 or 5?]
-  Mobile Development
-  BlockChain
-  Quality insurance/testing
-  UX designer
-  Front-end Web Development
-  Back-end Web Development
-  Fullstack Web Development [maybe redundant if multiple choices]
-  Game development
-  Application development
-  Machine Learning [include all forms of AI here?]
-  Network architecture, installation and/or administration
-  System analyst
-  Internet of Things
-  Embedded systems
-  Databases (administration and/or development)
-  Data analytics, data sciences
-  DevOps [include cloud computing/cloud-native here?]
-  DevRel [include tech writing/blogging/vlogging here?]
-  Customer Support
-  InfoSec
-  Other (specify)

Anwer here comes from my head, but also from [indeed](https://www.indeed.com/career-advice/finding-a-job/types-of-it-jobs).
